From 268910fc8302bdc66cf9b69f39fb1a0e2622c20f Mon Sep 17 00:00:00 2001
From: Pino Toscano <ptoscano@redhat.com>
Date: Tue, 16 Dec 2014 15:52:32 +0100
Subject: [PATCH 080/102] sysprep: recommend use of --selinux-relabel

Add a small documentation paragraph in the "SELINUX" section
recommending the usage of --selinux-relabel for guests using SELinux.

(cherry picked from commit 1f7db85b30d27ccdbb674dc9ee111db6756b59ae)
---
 sysprep/virt-sysprep.pod | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/sysprep/virt-sysprep.pod b/sysprep/virt-sysprep.pod
index 1636770..70894cf 100755
--- a/sysprep/virt-sysprep.pod
+++ b/sysprep/virt-sysprep.pod
@@ -466,6 +466,11 @@ I<--selinux-relabel> option.
 You can disable relabelling entirely by supplying the
 I<--no-selinux-relabel> option.
 
+Please note that if your guest uses SELinux, and you are doing operations
+on it which might create new files or change existing ones, you are
+recommended to use I<--selinux-relabel>.  This will help in making sure
+that files have the right SELinux labels.
+
 =head1 WINDOWS 8
 
 Windows 8 "fast startup" can prevent virt-sysprep from working.
-- 
1.9.3

